templates/v2/analysis/render_report.py

```diff
@@ -71,9 +73,9 @@ def get_data(
     codelist_1_link = codelist_url_root + codelist_1_link
     codelist_2_link = codelist_url_root + codelist_2_link
 
-    top_5_1_path = "output/foo/joined/top_5_code_table_1.csv"
-    top_5_2_path = "output/foo/joined/top_5_code_table_2.csv"
-    summary_table_path = "output/foo/event_counts.json"
+    top_5_1_path = f"output/{request_id}/joined/top_5_code_table_1.csv"
+    top_5_2_path = f"output/{request_id}/joined/top_5_code_table_2.csv"
+    summary_table_path = f"output/{request_id}/event_counts.json"
     figure_paths = {
         "decile": "joined/deciles_chart_practice_rate_deciles.png",
```

Comment:
I was momentarily confused about why the figure paths were relative to the "output/{request_id}" directory but the other paths specified them and I think that's because the figure paths are used directly in the report. Is that right?

It might be helpful to move the `top_5_1_data`, `top_5_2_data` and `summary_table_data` vars above the figure paths so they are next to the relevant path variables.

Something to consider in future, but not right now, is how to separate out the fetching/collecting of data from the display of the data.

Reply:
Yes, this is a little confusing. I've reordered as you suggested and will make a note to try and separate the fetching/collecting at a later date